src/main/java/edu/greenblitz/pegasus/RobotMap.java

```diff
@@ -89,6 +90,9 @@ public static class PressureSensor {
 			}
 		}
 		
+		public static class DigitalInputMap {
+			public static final int MACRO_SWITCH = 0;
+		}
 		
 		public static class Swerve {
 			public static final double WHEEL_CIRC = 0.0517 * 2 * Math.PI; //very accurate right now
```

Comment:
no comments, your code should speak for itself!
```suggestion
			public static final double WHEEL_CIRC = 0.0517 * 2 * Math.PI; 
```